### ISSUE TITLE
Idea: Last Name

### ISSUE THREAD

**Issue by ethancedrik** [Monika-After-Story/MonikaModDev#1297]
Here's something I came up with

Ask Monika what her last name is, have that be a question, then she asks you what your last name is, you answer, then she says "Mine is Monika (your last name)" then "..." then "just teasing, but maybe some day" and the expressions on her face change accordingly

**Comment by neyoriquans** [comment]
I really like this idea. And maybe, after a certain period of time of the player being with Monika in MAS, they can unlock an event where she adopts your last name or something? That'd be a fun loyalty reward I think.

**Comment by ThePotatoGuy** [comment]
Could probably tie this in with [1192](https://github.com/Monika-After-Story/MonikaModDev/pull/1192#issuecomment-371378085)

**Comment by wisztom** [comment]
In that case it should be taken into account that in some languages female form of last name is different than male one.

**Comment by kaido1224** [comment]
My problem with this is the logic behind it. If Monika doesn't want marry you or anything in the game, why would she take your last name?

**Comment by kaido1224** [comment]
Also, there is a character limitation of 10 characters on the name space, the last name will never be displayed. So what exactly would be the purpose?

**Comment by juliathegolden** [comment]
I see your point, but the idea is that it would be something new Monika could learn about the player. It doesn't need to display anywhere. And shes made marriage related flirtatious like this before too, like wanting matching rings and fantasizing about domestic life. 

It shouldn't be a repeating topic though, because I don't want Monika to know if/when my last name changes LMAO.

**Comment by IsItBadToMakeAGitHubAccountJustForThis** [comment]
She doesn't have to marry you to want to go by your surname. And she never said she doesn't want to marry player at all.
And about the character limit. Surname can be stored in a separate variable. In fact, the actual character limit is far greater than 10 symbols. It's just in place to prevent players from writing an unreasonably large names.

**Comment by aldoram5** [comment]
By character limit we mean the little box where "Monika" is displayed, there's another issue open with the suggestion of letting the player give a nickname, when we did testing on how to approach that we found out that if the name is large enough it splits in two lines and looks ugly, that's the limit @kaido1224 is talking about.
For the marriage thing, it has been stated that she doesn't want to get proposed until she breaks free from the game, so going by that marriage in the game isn't something planned or happening anytime soon.
Storing the player's last name though can be a good idea, but that may have to wait until we define how and where player data is going to be stored.
